cxx-qt-build/src/lib.rs

```diff
@@ -268,43 +273,115 @@ impl CxxQtBuilder {
         self
     }
 
-    /// Choose to disable Qt support
-    ///
-    /// This will disable including cxx-qt-lib headers.
-    pub fn disable_qt(mut self) -> Self {
-        self.qt_enabled = false;
-        self
-    }
-
     /// Specify rust file paths to parse through the cxx-qt marco
     ///
     /// Currently the path should be relative to CARGO_MANIFEST_DIR
     pub fn file(mut self, rust_source: &'static str) -> Self {
         self.rust_sources.push(rust_source);
+        println!("cargo:rerun-if-changed={}", rust_source);
+        self
+    }
+
+    /// Link additional [Qt modules](https://doc.qt.io/qt-6/qtmodules.html).
+    /// Specify their names without the `Qt` prefix, for example `"Widgets"`.
+    /// Do *not* specify "Core" or "Gui"; these are always linked.
```

Comment:
Once qt_modules is a set then it doesn't matter if they specify the same ones twice.